lib/rules/max-empty-lines/index.js

```diff
@@ -23,47 +21,28 @@ const rule = function (max) {
     if (!validOptions) {
       return
     }
-
+    
+    // `//`-comments are need to be processed by postcss-scss
+    // To do that we have to use .toString()
     const rootString = root.toString()
```

Comment:
In fact, without `root.toString()`, we have potential incorrect position on non standard syntax (//-comments). Because output of `root.source.input.css` isn't same as `root.toString()` when you used non standard syntax. (PostCSS-SCSS converts `// single line comment` to `/* single line comment*/`)